<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTransmitter</class>
 <widget class="QDialog" name="NewTransmitter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Device name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deviceNameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Device ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="deviceIdEdit">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>254</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sampling frequency:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="freqEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Transmitter interval:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="tranint">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65534</number>
       </property>
       <property name="value">
        <number>31250</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Activity threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="actThresEdit">
       <property name="maximum">
        <number>254</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Inactivity threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="inactThresEdit">
       <property name="maximum">
        <number>254</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Active time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="actTimeEdit">
       <property name="maximum">
        <number>254</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Channel:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="channelEdit">
       <property name="minimum">
        <number>11</number>
       </property>
       <property name="maximum">
        <number>26</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewTransmitter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewTransmitter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
